src/content/admin/cloud-credentials/aws.mdx

```diff
@@ -0,0 +1,123 @@
+
+   <p align="center">
+     <Image
+       src={require("@site/static/img/amazon-ecr-oidc-config.png").default}
+       alt="OIDC configuration in admin general view"
+     />
+   </p>
+
+Traditionally, `AUDIENCE` is the client id of the requester. You'll only exchange token for these audiences. It is the `aud` field of the JWT payload.
+
+We recommend creating a different/dedicated audience for each okteto cluster and region used even if you use the same identity provider:
+
+```
+your-okteto-instance.com/112233445566.your-region.amazonaws.com
+```
+
+or the more compact format:
+
+```
+your-okteto-instance.com/your-region
+```
+
+## Step 2: Create the Role
+
+
+First create the role and allow it to access s3:
+
+```bash
+# ROLE_ARN is the OpenIDConnectProviderArn role name created in step 1
+ROLE_ARN=arn:aws:iam::112233445566:oidc-provider/container.googleapis.com/v1/projects/my-project-12345/locations/us-central1/clusters/development
+AUDIENCE=deploy.okteto.dev
+OKTETO_SERVICE_ACCOUNT=system:serviceaccount:okteto:okteto
+cat <<EOT > trust-policy.json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Federated": "${ROLE_ARN}"
+      },
+      "Action": "sts:AssumeRoleWithWebIdentity",
+      "Condition": {
+        "StringEquals": {
+          "container.googleapis.com/v1/projects/my-project-12345/locations/us-central1/clusters/development:aud": "${AUDIENCE}",
+          "container.googleapis.com/v1/projects/my-project-12345/locations/us-central1/clusters/development:sub": "${OKTETO_SERVICE_ACCOUNT}"
+        }
+      }
+    }
+  ]
+}
+EOT
+
+
+aws iam create-role --role-name okteto-credentials --assume-role-policy-document file://trust-policy.json
+{
+    "Role": {
+        "Path": "/",
+        "RoleName": "okteto-credentials",
+        "RoleId": "AR...",
+        "Arn": "arn:aws:iam::112233445566:role/okteto-credentials",
+        "CreateDate": "2024-06-10T15:04:05+00:00",
+        "AssumeRolePolicyDocument": {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Principal": {
+                        "Service": "s3.amazonaws.com"
+                    },
+                    "Action": "sts:AssumeRole"
+                }
+            ]
+        }
+    }
+}
+```
+
+Attach an s3 policy to read and write s3 buckets:
+
+
+```bash
+aws iam attach-role-policy --role-name my-private-registry --policy-arn arn:aws:iam::aws:policy/AmazonS3FullAccess
+```
+
+
+## Step 3: Configure the credentials in Okteto
+
+Enable AWS in the [Cloud Credentials view](index.mdx) with the following configuration:
+
+- **Role ARN**: The Role ARN from the previous step
+- **Region**: The AWS Region
+- **Audience**: The Audience used for the Identity Provider
+
+
+This will allow Okteto to access the desired AWS resources in `okteto deploy` and `okteto test` commands.
```

Comment:
and `okteto destroy`, right?

Reply:
I would add a simple about how to deploy an s3 bucket on `okteto deploy`, and delete it on `okteto destroy`.